### ISSUE TITLE
Clarification needed on addition of `mergeSchema` option

### ISSUE THREAD

**Issue by koutaro-masaki** [eddeee888/graphql-code-generator-plugins#316]
When I updated the version of this package from v0.8.0 to v0.10.2, I noticed that a new file `schema.generated.graphqls` was being generated, which caused some confusion. I checked the release logs, but there was no mention of this update in any of the versions (I found the explanation only after searching the entire repository by the generated file name and eventually discovering an addition in the README). What is the purpose of this change? Will maintaining the previous behavior (i.e., not generating anything) affect the other behaviors of this package?

**Comment by eddeee888** [comment]
Hi @koutaro-masaki ,
Apologies for this confusion! That was an oversight on my part.
 This was added in this [PR](https://github.com/eddeee888/graphql-code-generator-plugins/pull/307) and released in v0.10.0. I have updated the release note. 

This merged schema is useful if in a few scenarios:
- If users have separate repo/s that need the schema, it's easier to share one file, instead of multiple files.
- Some tools may only support a single schema file instead of glob patterns e.g. `**/*.graphqls`, so a merged schema file makes it easier to integrate with these tools

If you don't need this file, feel free to turn it off with `mergeSchema: false`. No other functionalities need this file for now (and there's no plan to use this file for future functionalities right now)
